tests/acceptance/helpers/userSettings.js

```diff
@@ -54,52 +54,116 @@ module.exports = {
       email: 'sharee1@example.org'
     }
   },
+  createdUsers: {},
+
+  /**
+   *
+   * @param {string} userId
+   * @param {string} password
+   * @param {string} displayname
+   * @param {string} email
+   */
+  addUserToCreatedUsersList: function (userId, password, displayname = null, email = null) {
+    this.createdUsers[userId] = { password: password, displayname: displayname, email: email }
+  },
+
   /**
+   * gets the password of a previously created user
+   * if the user was not created yet, gets the password from the default Users list
+   * if the user is not in that list ether, returns the userId as password
    *
    * @param {string} userId
    * @returns {string}
    */
   getPasswordForUser: function (userId) {
-    if (userId in this.defaultUsers) {
+    if (userId in this.createdUsers) {
+      return this.createdUsers[userId].password
+    } else if (userId in this.defaultUsers) {
       return this.defaultUsers[userId].password
     } else {
-      return this.defaultUsers.regularuser.password
+      // user was not created yet and is not in the default users list, let the userId be the password
+      return userId
     }
   },
   /**
+   * gets the display name of a previously created user
+   * if the user was not created yet, gets the display name from the default Users list
+   * if the user is not in that list ether, returns the userId as display name
    *
    * @param {string} userId
-   * @returns {null|string}
+   * @returns {string}
    */
   getDisplayNameForUser: function (userId) {
+    let user = {}
+    if (userId in this.createdUsers) {
+      user = this.createdUsers[userId]
+    } else if (userId in this.defaultUsers) {
+      user = this.defaultUsers[userId]
+    } else {
+      return null
```

Comment:
The method doc removed `null` as a possible return value. Maybe:
```suggestion
      return userId
```